constants/vars.js

```diff
@@ -17,5 +17,5 @@ export const ROLE_OPTIONS = [
   { value: 'member', label: 'Member' },
 ]
 export const VALIDATION_EMAIL_REGEX = new RegExp(
-  /[a-z0-9]+@[a-z]+\.[a-z]{2,3}/g
+  /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/g
```

Comment:
do we want to restrict the tld from 2-4 characters? Also, I took a look at what some other libraries use for email validation in case we want to adopt a fairly well used version (in case we're not already doing that)

* Devise, a popular authentication library for Rails apps uses `/\A[^@\s]+@[^@\s]+\z/`
* The Ruby URI library uses 

```
/\A[a-zA-Z0-9.!\#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*\z/
```

Reply:
This is what they had in the app previously, so it's difficult to know why they made these decision. I feel like once we cut over to the latest we can use the yupresolver and not offload the validations to an actual library that handles it. 
https://react-hook-form.com/advanced-usage#ConnectForm:~:text=%7D-,Custom%20Hook%20with%20Resolver,-You%20can%20build